src/io/flutter/actions/OpenInAndroidStudioAction.java

```diff
@@ -125,19 +125,60 @@ private static VirtualFile getProjectForFile(@Nullable VirtualFile file) {
   }
 
   private static boolean isAndroidWithApp(@NotNull VirtualFile file) {
-    return file.getName().equals("android") && file.findChild("app") != null;
+    return file.getName().equals("android") && (file.findChild("app") != null || file.findChild("src") != null);
```

Comment:
Can you add some text describing the project structure we're looking for? This is the android/ subdir, with both app/ and src/ subdirs (flutter apps but not flutter plugins)?